src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -150,6 +150,12 @@ void RendererOpenGL::SwapBuffers() {
     {
         auto aggregator = Common::Profiling::GetTimingResultsAggregator();
         aggregator->AddFrame(profiler.GetPreviousFrameResults());
+        if (VideoCore::g_toggle_framelimit_enabled) {
+            Common::Profiling::Duration frame_time = aggregator->GetAggregatedResults().frame_time.avg;
+            using FloatMs = std::chrono::duration<float, std::chrono::milliseconds::period>;
```

Comment:
float average_frame_time = std::chrono::duration_cast<std::chrono::duration<float, std::chrono::milliseconds::period>>(frame_time).count();


with proper clang formatting of course.